lib/src/practice/quiz/practice_view.dart

```diff
@@ -0,0 +1,30 @@
+import "package:flutter/material.dart";
+import "package:go_router/go_router.dart";
+import "package:kana_to_kanji/src/core/widgets/app_scaffold.dart";
+import "package:kana_to_kanji/src/practice/quiz/practice_view_model.dart";
+import "package:stacked/stacked.dart";
+
+class PracticeView extends StatelessWidget {
+  static const routeName = "/practice";
+
+  const PracticeView({super.key});
+
+  @override
+  Widget build(BuildContext context) =>
+      ViewModelBuilder<PracticeViewModel>.reactive(
+          viewModelBuilder: () => PracticeViewModel(GoRouter.of(context)),
+          builder: (context, viewModel, child) => AppScaffold(
+              resizeToAvoidBottomInset: true,
+              showBottomBar: true,
+              appBar: AppBar(
+                title: SizedBox(
+                  width: double.infinity,
+                  child: FilledButton(
```

Comment:
The button should be on the `body` and the `appBar` left empty

Reply:
Updated